packages/home-service/src/resolver.ts

```diff
@@ -30,7 +30,7 @@ export const HomeResolver = {
       .then(response => {
         return Object.assign(response, args.input)
           .save()
-          .then((user: any) => user);
+          .then((data: any) => data);
```

Comment:
The query resolvers should fetch the user details from the User/Group Microservice and stitch them back in the response.

Also, try to use a single query to fetch all the users required for the response.

Reply:
@deshmukhmayur that would be done in a different PR https://github.com/1-Platform/one-platform/pull/284

Reply:
This is to add the owners array

Reply:
That PR is mainly for the timestamps. But I guess the code change is similar. Ok.